app/controllers/students/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Students::SessionsController < Devise::SessionsController
+ layout 'login_layout'
+before_filter :configure_sign_in_params, only: [:create]
+ # GET /resource/sign_in
+  def new
+     super
+   end
```

Comment:
Is this still required?  I guess I have told you to move this in to other controller.